### DISCUSSION TITLE
React navigation 7

### DISCUSSION THREAD

**Discussion by umardev500** [react-navigation/react-navigation#12118] [Q&A]
When it will be latest stable version guys?
is that ready for production use?

**Comment by export-mike** [comment]
should we expect to see web being supported?

**Reply by export-mike** [reply]
whats the support for react-navigation looking like? does react-navigation need some more funding, engineers? Support?

Where will react-navigation be in 1yr, 2years time? 

I ask because as an engineer working on an application with 300+ screens we're pretty baked into react-navigation

What should we be expecting in future releases?

1. web
2. deep linking url navigation
3. native performance

**Reply by satya164** [reply]
> does react-navigation need some more funding, engineers? Support

We need more people helping out with issue management, and looking into bug reports. If you or someone you know has spare time, I'm happy to give them issue management permissions for the repo.

> What should we be expecting in future releases?
> web
> deep linking url navigation
> native performance

I believe we already have all of these. Improving DX and improving docs is what I'm focusing on. Of course, there is always scope for improvement in many areas. If you have ideas, please post them in our discussions.

For now, the main goal is to release v7.

**Comment by satya164** [comment]
We're waiting for a few things, but in essence, the current RC is what will end up as the stable release. If you're waiting for a stable version number, you can already use the RC.
